tss-esapi/src/structures/nv/storage/public.rs

```diff
@@ -27,6 +27,7 @@ pub struct NvPublic {
 }
 
 impl NvPublic {
+    #[allow(unused_qualifications)]
```

Comment:
Does the compiler complain if you remove the "unused" qualifications?

Reply:
I am not sure. The call is specified using the full qualifications but because 1.80 puts the size_of in the preamble the qualifications do not get used and that is what it complains about.

I will test it.

Reply:
You cannot remove it completely because then you will get an error.
And there is no 'qualifications' lint so you cannot write ```#[allow(qualifications)]```.